client/deposits/list/index.tsx

```diff
@@ -146,6 +162,18 @@ export const DepositsList = (): JSX.Element => {
 					formatExplicitCurrency( deposit.amount, deposit.currency )
 				),
 			},
+			fees: {
+				value: formatExportAmount( deposit.fee, deposit.currency ),
```

Comment:
I'm less familiar with deposits, but I'd say that we should surface the total fees paid for every transaction included in the deposit. As it is now, we seem to surface the fee associated with the deposit. In my local (can be wrong) the `deposit.fee` is always zero but when I click in the row then I see all the transactions, each with their fee. I think those are the fees that we should surface.
<img width="980" height="675" alt="Screenshot 2025-11-03 at 18 13 54" src="https://github.com/user-attachments/assets/d2ee5166-0c86-4309-898a-7c306e227833" />

Reply:
Made the changes @htdat and @mgascam. Could you take a look again? Also changed 195184-ghe-Automattic/wpcom